### ISSUE TITLE
Insertion 10 minutes time penalty for player 

### ISSUE THREAD

**Issue by Chrissi2802** [SebastianSiedler/Mannschaftskasse#11]
Please include the 10-minute time penalty in the statistics. Currently only yellow and red cards are available.

**Comment by SebastianSiedler** [comment]
How do you want the 10min to be included? 
Can it just be one penalty per match? Is it always 10 minutes?

How do you want them to be shown in the statistics page? Total Minutes, Count, ...

Feel free to open a RP yourself 😄

**Comment by Chrissi2802** [comment]
Yes, the time penalty is always 10 minutes and can only be given once per player per game. 
I would record the number of time penalties in the statistics.
